woocommerce/woocommerce
# DO NOT MERGE: add version support to docs manifest

### Changes proposed in this Pull Request:

This PR adds the following to the manifest generator

- A manifest version that is retrieved from `plugins/woocommerce/woocommerce.php`
- front matter support for a `version: X.X.X` in `.md` files



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Edit `plugins/woocommerce/woocommerce.php`
2. Change the version in the plugin header to `8.8.0-beta`
3. Edit a non `README.md` markdown file under the docs folder
4. Add `version: 8.9.0` to the front matter at the top of the file
5. Run `pnpm utils md-docs docs woocommerce` from the root of the repo
6. Check `manifest.json` for
- the manifest `version` (at the end of the file) is `8.8.0-beta`
- the manifest entry for the `.md` file you edited contains the version you added



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
